src-ts/proposalPipeline.ts

```diff
@@ -109,6 +111,8 @@ export class StageTracker extends EventEmitter {
               status === ProposalStageStatus.EXECUTED
                 ? await this.stage.getExecutionUrl()
                 : undefined,
+                // @ts-ignore
+              description: this.stage.description || undefined
```

Comment:
Could maybe do `this.stage instanceof GovernorQueueStage ? this.stage.description : undefined`, then you wont need to ts-ignore. Could also call the property on this event `proposalDescription` in case we add a different description elsewhere.